### ISSUE TITLE
The Elfa mutate "Very little sleep and tireless" are not fit well with weariness

### ISSUE THREAD

**Issue by vipches** [CleverRaven/Cataclysm-DDA#48232]
as we know, to make weariness recovering, you had to sleep in order to recovering your weariness, but here's problem, because those mutates, making you not tired enough to sleep, and that means you can't recovering your weariness quickly, and that will be a problem if we wanna have those mutates, we need a solution to deal with this problem.

**Comment by actual-nh** [comment]
You don't have to sleep to recover weariness - but you recover it twice as fast when you're asleep. The original weariness proposal did include something re this - either make you recover much faster when asleep, or recover faster without sleeping - but it seems to have gotten lost. Once #47653 is in, it will be (much) easier to fix this, BTW.

**Comment by Dhanvanthri** [comment]
I want to add on to this that debug protection doesn't protect you from weariness, and there's no way to change weariness using the debug menu.
I've been testing mutations, and this was a major problem.

**Comment by actual-nh** [comment]
> I want to add on to this that debug protection doesn't protect you from weariness, and there's no way to change weariness using the debug menu.
> I've been testing mutations, and this was a major problem.

Good point. I haven't done anything with the programming for the debug menu (and it would have to go into 0.F-dev, not master, almost certainly). Debug protection being extended to cover weariness - I can see arguments for and against (e.g., working on stamina/weariness from fighting).

**Comment by I-am-Erk** [comment]
If we have not already fixed this, we really do need the "less sleep" mutations to increase the amount of weariness you recover while at NO_ACTIVITY, with "no sleep" increasing it to almost as high as baseline characters get from sleep.

**Comment by actual-nh** [comment]
Yes, it still needs fixing. I have in mind SLEEP_ACTIVITY being a lower level than NO_ACTIVITY for purposes of feeding into weariness on potentially something other than a strict 5-minute timescale.
